HomeSol/Home_5/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Home_5;
+using System;
+using System.Text.RegularExpressions;
+
+namespace MyApp // Note: actual namespace depends on the project name.
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            
+
+            Student[] group1 = new Student[5]; // Generating group 1
+            for (int i = 0; i <= group1.Length - 1; i++)
+            {
+                int stGroup = 1;
+                int id = 100 + i;
+                Random random = new Random();
+                string[] names = { "Francis", "Caroline", "Harold", "Alise", "Emily" };
+                string name = names[i];
+                group1[i] = new Student(id, name, (int)random.NextInt64(17, 25), stGroup);
```

Comment:
Try to avoid magic numbers. Use variables instead. 
int someVariableWithPurpose = 17;